src/particles/wakefields/WakeConvolution.cpp

```diff
@@ -0,0 +1,152 @@
+#include "WakeConvolution.H"
+#include "particles/ImpactXParticleContainer.H" //Includes all necessary AMReX headers
+#include "initialization/InitDistribution.H"
+
+#ifdef ImpactX_USE_FFT
+#include <fftw3.h> //Fastest Fourier Transform in the West
```

Comment:
This line causes a problem during CI testing.  Now that access to FFTW is enabled elsewhere, it's not clear if this line is needed, or how it should be modified.  Suggest trying without this line.

Reply:
Removing this line causes compilation errors locally for unknown type names associated with the FFTW variable types.

Reply:
Understood.  It looks like this is the main problem left.  Only 3 of the tests are failing, and in 2 cases this is because the fftw header file isn't located during the build.  We may need to consult with Axel on this part.

Reply:
I will help here. On GPU (CUDA/HIP), we will use GPU-accelerated FFTs instead of FFTW. We have already wrappers in ABLASTR for this, which we can use here, too.

Reply:
First updates pushed today.
We will pull in an update to ABLASTR/WarpX here once this PR is merged: https://github.com/ECP-WarpX/WarpX/pull/5112